validate/validate.go

```diff
@@ -503,8 +503,9 @@ func (v *Validator) CheckLinux() (msgs []string) {
 	case "rslave":
```

Comment:
On the ` `, `rprivate` ,`rslave`, `rshared` these values, I am not sure whether to delete, because these values are not in the `spec`.